plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/data/ExecuteInsertBatchImpl.java

```diff
@@ -110,6 +114,11 @@ StringBuilder prepareQueryForStatement(
         DBSTable table,
         boolean useMultiRowInsert,
         Map<String, Object> options) throws DBCException {
+        
+        Assert.isLegal(attributes.length == handlers.length);
+        if (!useMultiRowInsert) {
+            Assert.isLegal(attributes.length == attributeValues.length);
+        }
```

Comment:
```suggestion
        Assert.isLegal(useMultiRowInsert || attributes.length == attributeValues.length);
```

Either we use multi row insert and don't care about this assumption, otherwise we use "single row insert" and length must be the same :)

Reply:
Done!